.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,34 @@
 <!---
+
 Thanks for creating a pull request!
 
-Please start your PR title with the appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+## TODO
 
-chore: [your title]
-docs: [your title]
-feat: [your title]
-fix: [your title]
-refactor: [your title]
-rfc: [your title]
-test: [your title]
--->
+ - 🚧 Documentation in README, crate, module, function and/or other locations added/adapted.
```

Comment:
Aren't cargo-docs in crates, modules, functions etc mandatory to pass the CI?
Maybe we should list here only these requireme ts that are not covered by the CI.

Reply:
CI cannot check whether some code-change requires a change to the documentation. 

At least for me, this can also be a helpful reminder to add stuff I might have forgotten about before creating a PR with CI immediately yelling at me.

Reply:
We don't (yet) have semantic analyses in CI to determine whether the docs have been updated, or check any non-API documentation.

Reply:
Okay fair, so this point mostly stands for keeping documentation up-to-date rather than adding it (missing documentation do triggers a warning).

I believe the following form make the sentence simpler:
```suggestion
 - 🚧 Documentation is added and up-to-date in README, crate, module, function and/or other locations.
```

Reply:
For me personally, the current phrasing is easier to read.

Reply:
Then it's my bad English that triggers something in me when I read the action at the end XD

Feel free to mark this as resolved if all of you disagree on my changes